src/pyff/utils.py

```diff
@@ -273,7 +273,13 @@ def load_url(url, enable_cache=True, timeout=60):
     if url.startswith('file://'):
         path = url[7:]
         if not os.path.exists(path):
-            raise IOError("file not found: %s" % path)
+            log.error("file not found: %s" % path)
+            return _Resource(result=None,
```

Comment:
The problem is that we already have an outer retry. The inner retry introduces a failure mode when (if you say loose network) can fail to download a feed but the instrumentation in pyffd doesn't have any record of that feed. Your approach with "empty" Resource objects allows monitoring of complete failures.